src/main/webapp/app/exercises/programming/assess/programming-assessment-dashboard/programming-assessment-dashboard.module.ts

```diff
@@ -0,0 +1,27 @@
+import { NgModule } from '@angular/core';
+import { ProgrammingAssessmentDashboardComponent } from 'app/exercises/programming/assess/programming-assessment-dashboard/programming-assessment-dashboard.component';
+import { ArtemisSharedModule } from 'app/shared/shared.module';
+import { FormDateTimePickerModule } from 'app/shared/date-time-picker/date-time-picker.module';
+import { FeatureToggleModule } from 'app/shared/feature-toggle/feature-toggle.module';
+import { AssessmentInstructionsModule } from 'app/assessment/assessment-instructions/assessment-instructions.module';
+import { FormsModule } from '@angular/forms';
+import { RouterModule } from '@angular/router';
+import { ArtemisResultModule } from 'app/exercises/shared/result/result.module';
+import { ArtemisAssessmentSharedModule } from 'app/assessment/assessment-shared.module';
+
+@NgModule({
+    imports: [
+        ArtemisSharedModule,
+        FormDateTimePickerModule,
+        FormsModule,
+        FeatureToggleModule,
+        AssessmentInstructionsModule,
+        RouterModule.forChild([]),
+        ArtemisResultModule,
+        ArtemisAssessmentSharedModule,
+    ],
+    declarations: [ProgrammingAssessmentDashboardComponent],
+    entryComponents: [],
```

Comment:
These empty arrays are not necessary.

Reply:
I removed those too, thanks for the tip :D